x/downtime-detector/types/constants.go

```diff
@@ -1,9 +1,81 @@
 package types
 
+import (
+	fmt "fmt"
+	"strings"
+	time "time"
+
+	"github.com/tidwall/btree"
+)
+
 const (
 	ModuleName = "downtime-detector"
 	StoreKey   = ModuleName
-	RouterKey  = ModuleName
+	// must be alphanumeric
+	RouterKey = "downtimedetector"
```

Comment:
I think we made concentratedliquidity just not use the "-" in the module name to prevent the diff here, might prevent some confusion down the road.

Reply:
good idea